addons/godot-next/resources/resource_collections/resource_collection.gd

```diff
@@ -55,13 +64,36 @@ func _add_element(p_script: Script) -> void:
 func _refresh_data() -> void:
 	assert false
 
+# Export properties within the 'data' group
+func _export_data_group() -> Array:
+	return [ PropertyInfo.new_editor_only(DATA_PREFIX + "dropdown").to_dict() ]
+
+# Export properties within the 'setup' group
+func _export_setup_group() -> Array:
+	return [ PropertyInfo.new_resource(SETUP_PREFIX + "base_type", "Script").to_dict() ] if not _type_readonly else []
+
+# Injects controls to the 'EditorInspectorPlugin'
+func _parse_property(p_plugin: EditorInspectorPlugin, p_pinfo: PropertyInfo) -> bool:
+	match _parse_path(p_pinfo.name, DATA_PREFIX):
+		"dropdown":
+			var elements = _find_inheritors()
+			var control = InspectorControls.new_dropdown_selector(elements, self, "_on_dropdown_selector_selected")
+			p_plugin.add_custom_control(control)
+			return true
+	return false
+
 ##### PUBLIC METHODS #####
 
 func clear() -> void:
 	assert false
 
 ##### PRIVATE METHODS #####
 
+func _parse_path(p_path: String, p_prefix: String) -> String:
```

Comment:
Maybe change the name from `_parse_path` to `_extract_name_from_path` or something, to prevent any potential confusion with the other `_parse_*` methods.